python/py-idna-ssl/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-idna-ssl
+version             1.0.1
+categories-append   devel
+platforms           darwin
+license             MIT
+
+python.versions     34 35 36
+
+maintainers         {ipglider.org:miguel @ipglider} openmaintainer
+
+description         Patch ssl.match_hostname for Unicode(idna) domains support
+long_description    Patch ssl.match_hostname for Unicode(idna) domains support
```

Comment:
It would probably be better to have a long description that described the port a bit better. I have no idea what it does given the current one.

Reply:
BTW, having looked at https://pypi.org/project/idna_ssl/ I still have no idea what this does. (The description there is, as I'm sure you know, identical...)

Reply:
The Python library [idna](https://github.com/kjd/idna) does not work as expected when using SSL under some versions of the interpreter.

[aiohttp](https://github.com/aio-libs/aiohttp) depends on [idna-ssl](https://github.com/aio-libs/idna-ssl) to solve this problem. More info in https://github.com/aio-libs/aiohttp/issues/949.

I will change the description to be more useful. Thank you for taking the time to review my PRs.